liams-news/src/Components/ Comments/Components/Comments.jsx

```diff
@@ -0,0 +1,33 @@
+import axios from "axios"
```

Comment:
The name of this file is a little confusing - "Components/Comments/Components/Comments"